src/components/red-header.js

```diff
@@ -3,7 +3,7 @@ import React, { useEffect, useState } from 'react';
 import { Link } from 'gatsby';
 import whiteLogo from '../files/icons/CA-white.svg';
 import redLogo from '../files/icons/ca-extended-logo.svg';
-import menuIconWhite from '../files/icons/menu-icon-white.svg';
+import menuIcon from '../files/icons/menu-icon.svg';
```

Comment:
@jaube05 I created this ticket: https://civicactions-home.atlassian.net/browse/HOMER20-521 to change how we place the mobile menu icon to an inline svg instead of an <img>. 
That way we can still use a white icon in the red header but change the svg fill to black in mobile and in the sticky header.